### PR TITLE
Dependency upgrades

### PR DESCRIPTION
**Pull request by ljacomet**
* Upgrade AWS S3 libraries
* Upgrade Jackson

### CODE REVIEW

**Comment by ljacomet** [comment]
@bot-gradle test this

File: gradle/verification-metadata.xml [hunk 1]
```diff
@@ -85,6 +86,10 @@
          <trusted-key id="1fa868a348719e88b6d0de24c03ef1d7d692bcff" group="org.scala-lang"/>
          <trusted-key id="21f0942275ea159a501d375328852008a47bb995" group="^com[.]google($|([.].*))" regex="true"/>
          <trusted-key id="253e8e4c6fb28d11748115c1249dee8e2c07a0a2" group="com.github.javaparser" name="javaparser-core"/>
+         <trusted-key id="28118c070cb22a0175a2e8d43d12ca2ac19f3181">
+            <trusting group="com.fasterxml.jackson.core"/>
+            <trusting group="^com[.]fasterxml[.]jackson($|([.].*))" regex="true"/>
```
Comment (eskatos) [inline gradle/verification-metadata.xml:91]: :question: Doesn't the former implies the latter?

**Review by eskatos — APPROVED** [review]

File: gradle/verification-metadata.xml (hunk 1 above)
Comment (ljacomet) [reply gradle/verification-metadata.xml:91]: Did not think to hard about that one at this point tbh

**Comment by ljacomet** [comment]
@bot-gradle test and merge

**Pull request merged**